experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -216,7 +216,23 @@ <h4>Audience</h4>
             </tr>
             <tr>
               <th>Recipe JSON</th>
-              <td colspan="3">{{ experiment.targeting }}</td>
+              <td colspan="3">
+                <div class="collapse" id="collapseRecipe">
```

Comment:
Giving this div a max-height and overflow scroll might be a nicer experience for huge recipes.

Reply:
Maybe, but we can save that for a followup if necessary.